README.md

```diff
@@ -14,8 +14,59 @@ the packages individually, for example:
     make -C debs/xenial/archivematica
     make -C debs/xenial/archivematica-storage-service
 
-### Test packages
+#### Using parameters
 
+Most makefiles support parameters in the format of environment variables. They 
+may be a bit different between packages, but the most common are:
+    BRANCH
+    VERSION
+    RELEASE
+
+So in order to build an specific branch / version, this command can be used from
```

Comment:
Typo: "an specific" should be "a specific".

Reply:
Done